compiler/Cargo.lock

```diff
@@ -429,7 +429,6 @@ dependencies = [
  "graphql-ir",
  "graphql-syntax",
  "intern",
- "relay-config",
```

Comment:
this seems wrong, should this change be reverted? Might have been an artifact of the other branch I was working on

Reply:
Oh yeah, our cargo.lock is out of date. I'll import https://github.com/facebook/relay/pull/4747 which should fix it. in the mean time you can revert or leave this, either way is fine.

Reply:
This has landed on main now. Should be able to rebase.